### DISCUSSION TITLE
PINTEREST Part 4 Create - New boards in Firebase but don't show to DOM

### DISCUSSION THREAD

**Discussion by jimguistical** [nss-evening-cohort-14/evening-client-side#171] [Help Me!] [answered]
Jim Conner

Other help tickets: no

Issue: Form for creating boards successfully shows new boards in FB, but the new cards won't show up on DOM at form submit.

What I have tried:
- tried using console.warn on boardId
- tried editing and then commenting out selectBoards
- Created boards now add to firebase but aren't showing to dom
Google links:
https://stackoverflow.com/questions/45646155/user-input-not-writing-to-firebase
https://stackoverflow.com/questions/60746406/form-submission-not-showing-up-in-firebase

Links to PR:
https://github.com/jim-conner/ASSIGNMENT-Firebase-Pinterest
https://github.com/jim-conner/ASSIGNMENT-Firebase-Pinterest/tree/part4-create
https://github.com/jim-conner/ASSIGNMENT-Firebase-Pinterest/blob/part4-create/src/javascripts/events/domEvents.js
https://github.com/jim-conner/ASSIGNMENT-Firebase-Pinterest/blob/part4-create/src/javascripts/components/forms/addBoardForm.js

**Comment by TrinityChristiana** [comment]
It does not look like you're adding the user id to the board info when you create a board. Are you?

**Reply by TrinityChristiana** [reply]
Looks like on these lines only the first name and image are being pushed to FB: https://github.com/jim-conner/ASSIGNMENT-Firebase-Pinterest/blob/part4-create/src/javascripts/events/domEvents.js#L32-L36

**Reply by jimguistical** [reply]
Confirming I'm passing uid along with boardObject as a parameter for createBoard. 
For last name, I just left it out for now (I think I actually want the boards to have titles and not FN/LN later on).  The form still posts to FB correctly.
![image](https://user-images.githubusercontent.com/76793747/110561139-c767ee80-810c-11eb-93dc-34a74f75b525.png)

**Comment by jimguistical** [comment, marked as answer]
I was not passing "uid" as an argument when calling createBoard on my domEvents -- createBoard needed both boardObject and uid.

Takeaway: Don't be afraid to console.warn whole fb urls, especially if you see "undefined" come back.
![image](https://user-images.githubusercontent.com/76793747/110563786-73133d80-8111-11eb-8788-58dd6871be43.png)
